logstash-core/lib/logstash/api/modules/base.rb

```diff
@@ -0,0 +1,41 @@
+require "logstash/api/app_helpers"
+require "logstash/api/command_factory"
+
+module LogStash
+  module Api
+    module Modules
+      class Base < ::Sinatra::Base
```

Comment:
well, in that case for me (in my head) this is more like a utility base class, right? so ... let me explain. by having this and the actual resources in separate location we could achieve a more clear separation of concerns, right? I say that because end of the day, there is no actual resource setup with this class. What do you think?